modules/sdk-coin-zec/README.md

```diff
@@ -0,0 +1,39 @@
+# BitGo sdk-coin-zec
+
+SDK coins provide a modular approach to a monolithic architecture. This and all BitGoJS SDK coins allow developers to use only the coins needed for a given project.
+
+## Installation
+
+All coins are loaded traditionally through the `bitgo` package. If you are using coins individually, you will be accessing the coin via the `@bitgo/sdk-api` package.
+
+In your project install both `@bitgo/sdk-api` and `@bitgo/sdk-coin-zec`.
+
+```shell
+npm i @bitgo/sdk-api @bitgo/sdk-coin-zec
+```
+
+Next, you will be able to initialize an instance of "bitgo" through `@bitgo/sdk-api` instead of via `bitgo`.
+
+```javascript
+import { BitGoAPI } from '@bitgo/sdk-api';
+import { Zec, Tzec } from '@bitgo/sdk-coin-zec';
+
+function init() {
+  const bitgoapi = new BitGoAPI({ env: 'test' });
+  bitgoapi.register('zec', Zec.createInstance);
+  bitgoapi.register('tzec', Tzec.createInstance);
+  return bitgoapi;
```

Comment:
Can't we write a more convenient interface like this?
```suggestion
  return new BitGoAPI({ env: 'test' }).withCoins(Zec, Tzec);
```

Reply:
I like the idea that we could get the symbol information from the coin. Can probably just keep `.register` and change it to accept one or many constructors.